oauthlib/oauth1/rfc5849/signature.py

```diff
@@ -280,11 +259,11 @@ def collect_parameters(uri_query='', body=[], headers=None,
     #    `RFC3986, Section 3.4`_.  The query component is parsed into a list
     #    of name/value pairs by treating it as an
     #    "application/x-www-form-urlencoded" string, separating the names
-    #    and values and decoding them as defined by
-    #    `W3C.REC-html40-19980424`_, Section 17.13.4.
+    #    and values and decoding them as defined by W3C.REC-html40-19980424
+    #    `W3C-HTML-4.0`_, Section 17.13.4.
```

Comment:
why did you change this?

Reply:
The shorter reference identifier is more easily recognisable as "HTML 4.0". The "-40-19980424" makes it harder for the human reader and adds little value in this context. Also, to satisfy PEP-8 maximum line length rule.